appyters/RNA_Level_Functional_Predictions/templates/fields/AutocompleteField.j2

```diff
@@ -0,0 +1,6 @@
+<div>
+    <p>Search RNA gene symbol:</p>
+    <div class="autocomplete" style="width:300px;margin-bottom:20px"> 
```

Comment:
Please ensure your field works well on different sized displays. I would recommend utilizing bootstrap classes which allow you to quickly size it based on the display.

Reply:
@cyoon23 this is the only thing left. Please use bootstrap grid classes, for example:

`<div class="autocomplete col-sm-12 col-lg-6 mb-2">`

`mb-2` is a "2x spacer" bottom margin https://getbootstrap.com/docs/4.4/utilities/spacing/
`col-sm-12` means it should take the full width on small displays
`col-lg-6` means it should take half the width on large displays
https://getbootstrap.com/docs/4.4/layout/grid/#grid-options